test/framework/mesh/tests.json

```diff
@@ -27,5 +27,24 @@
         "key" : "Exporting mesh to VTK files with base new_bnd_ids"
       }
     ]
+  },
+  {
+    "file": "volume_per_material.lua",
+    "comment": "2D test to compute and display volume per material ID.",
+    "num_procs": 3,
+    "checks": [
+      {
+        "type": "KeyValuePair",
+        "key": "[0]  Material ID: 0 Volume:",
+        "goldvalue": 12.5,
+        "abs_tol": 1.0e-6
+      },
+      {
+        "type": "KeyValuePair",
+        "key": "[0]  Material ID: 1 Volume:",
+        "goldvalue": 12.5,
```

Comment:
It would make the test more robust if it was set up such that volume of material 1 was different than volume of material 0.

Reply:
done